app/controllers/initiatives_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class InitiativesController < ApplicationController
+  before_action :set_initiative, only: %i[show destroy update]
+
+  def index
+    render json: Initiative.all.as_json(include: {
+                                          owner: { only: [:id] },
+                                          parent: { only: [:id] }
+                                        })
+  end
+
+  def show
+    render json: @initiative.as_json(include: {
+                                       owner: { only: [:id] },
+                                       parent: { only: [:id] }
+                                     })
```

Comment:
If you want to return something super custom you can use Jbuilder

Reply:
Oh, I didn't know about that! Implemented!